docs/XBlocks.md

```diff
@@ -0,0 +1,217 @@
+    student progress of reviewing other teammates by displaying completion icons next to each teammate.
+    * No author-customizable settings
+    
+    ![Teammate selector screenshot](/docs/images/stage_components/teammate_selector.png)
+    
+* `Group Selector` - displays a list of groups and allows switching between them. Only usable in conjunction with
+    `Peer Grading` stage and `Review Question` blocks. When group is selected, `Peer Grading` stage downloads 
+    current student's responses to `Review Question` and sets corresponding `Review Question` values. Displays current
+    student progress of reviewing other teammates by displaying completion icons next to each teammate. Allows reviewing
+    group submissions by clicking "View team submissions"
+    * No author-customizable settings
+    
+    ![Group selector screenshot](/docs/images/stage_components/group_selector.png)
+    
+* `Review Question` - represents single review question.
+    * `Question ID` - question identifier. Defaults to automatically generated random string
+    * `Question Text` - question text
+    * `Assessment Question Text` - this setting can be used to override `Question Text` when answers displayed for 
+        evaluation. E.g. `Question Text`: "Was it easy to communicate wit this teammate?" can be overridden with 
+        `Assessment Question Text`: "Was it easy to communicate with you?".
+    * `Question Content` - question content. HTML markup that must contain some HTML input control (`input`, 
+        `select` or `textarea`). The value of the control becomes the response to the question.
+    * `Required` - boolean flag. If True, the question is required - it is enforced to be not empty in UI and is 
+        used in stage state calculation.
+    * `Grading` - boolean flag. If True, this question is a grading question - it should provide answer as a decimal
+        number and it is used for grading.
+    * `Single Line` - boolean flag. If set to True, question label and content are placed on the same horizontal 
+        line, allowing for more compact layout.
+    * `CSS Classes` - author-defined additional CSS classes.
+
+    ![Review questions screenshot](/docs/images/stage_components/review_questions.png)
+    
+* `Team Evaluation Display`, `Grade Evaluation Display` - blocks to display review question responses in "display" 
+    stages (`Evaluation Display` and `Grade Display`, respectively)
+    * Settings:
+        * `Question ID` - question identifier - responses to this question will be shown.
+        * `Show Mean Value` - boolean flag, alters question response representation. If True the response will be 
+            considered numerical, and a mean value of all responses will be displayed.
+            
+    ![Review feedback screenshot](/docs/images/stage_components/display_blocks.png)
+            
+* `Project Team` - this block displays current project team - all student teammembers (including himself) and allows
```

Comment:
@e-kolpakov "- all student teammembers (including himself)" ⟶ "(all student team members including the current user)"